skimage/data/__init__.py

```diff
@@ -261,3 +262,36 @@ def rocket():
 
     """
     return load("rocket.jpg")
+
+
+def hdr_images():
+    """Bracket images taken from the UNIS building on Svalbard, overlooking the
+    Hjorthfjellet mountain
+
+    These bracketed images are useful for showing the application of High
+    Dynamic Range (HDR) imaging.
+
```

Comment:
I suggest to replace all the lines above by:


Bracket images useful for High Dynamic Range (HDR) imaging.

Images taken from the UNIS building on Svalbard, overlooking the Hjorthfjellet mountain.